test/e2e/framework/helpers/k8s/k8s_utils.go

```diff
@@ -391,7 +451,38 @@ func (k *KubeCtl) DeleteNamespace(namespace string) error {
 }
 
 func (k *KubeCtl) TearDownNamespace(namespace string) error {
-	err := k.DeletePods(namespace)
+	var err error
+	err = k.DeleteDeployments(namespace)
```

Comment:
@chenyulin0719 Could you add comments to explain the order of delete?

Reply:
I think this is fairly self-explanatory. Objects that create other objects are deleted first.